package.json

```diff
@@ -46,6 +46,8 @@
     "d3-time-format": "^2.0.0",
     "element-resize-detector": "^1.1.6",
     "lodash": "^4.11.0",
+    "moment": "^2.17.1",
```

Comment:
you're right, we can probably do away with the `moment` dependency now that we're depending on `react-day-picker` for doing the hard work.